release-please-config.json

```diff
@@ -7,17 +7,17 @@
     ".": { "release-as": "" },
     "packages/api-explorer": { "release-as": "" },
     "packages/code-editor": { "release-as": "" },
-    "packages/extension-api-explorer": { },
-    "packages/extension-sdk": { },
-    "packages/extension-sdk-react": { },
+    "packages/extension-api-explorer": { "release-as": "" },
+    "packages/extension-sdk": { "release-as": "" },
+    "packages/extension-sdk-react": { "release-as": "" },
     "packages/extension-utils": { "release-as": "" },
-    "packages/hackathon": { },
+    "packages/hackathon": { "release-as": "" },
     "packages/run-it": { "release-as": "" },
-    "packages/sdk": { },
+    "packages/sdk": { "release-as": "" },
     "packages/sdk-codegen": { "release-as": "" },
     "packages/sdk-codegen-scripts": { "release-as": "" },
     "packages/sdk-codegen-utils": { "release-as": ""},
-    "packages/sdk-node": { },
+    "packages/sdk-node": { "release-as": "" },
```

Comment:
I don't think we want any of these - `"release-as": ""` means use semver - no config means inherit from above (L5)

Reply:
No problem. I'll revert. I wanted to make it consistent in case that is why we saw the strangeness in the version number generation.